mava/configs/env/scenario/con-10x10x5a.yaml

```diff
@@ -0,0 +1,5 @@
+task_name: con-10x10x5a
+
```

Comment:
To all the scenario configs can we please add a `name` attribute which is the env name used for initialisation and then use that to create the envs please? Similar to other envs. 
Similar for all other scenario configs as well please.